orchestrators/IntelligenceOperations/src/DTOs/AiCapabilityStatus.php

```
@@ -0,0 +1,105 @@
+        public bool $operatorCritical,
+        public array $diagnostics = [],
+    ) {
+        $this->assertNonEmptyString($this->featureKey, 'feature key');
+        $this->assertNonEmptyString($this->messageKey, 'message key');
+        AiStatusSchema::assertCapabilityGroup($this->capabilityGroup);
+        AiStatusSchema::assertEndpointGroup($this->endpointGroup);
+        AiStatusSchema::assertFallbackUiMode($this->fallbackUiMode);
+        $this->assertReasonCodes($this->reasonCodes);
+        $this->assertDiagnostics($this->diagnostics);
+        $this->assertStatusAvailabilityPair($this->status, $this->available);
+    }
```

PHPMD (2.15.0)

[warning] The method __construct has 10 parameters. Consider reducing the number of parameters to less than 10. (undefined)

(ExcessiveParameterList)